include/arch/arm/cortex_m/mpu/arm_mpu_v8m.h

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2018 Linaro Limited.
+ * Copyright (c) 2018 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Privileged No Access, Unprivileged No Access */
+//#define NO_ACCESS       0x0
```

Comment:
These access permissions configuration is not applicable for ARMv8-M. I kept the commented-out code, to make it visible to the reviewing process.